generate_bq2_queries.qmd

```diff
@@ -218,7 +222,8 @@ for (ith_table in q_nopii) {
         AND p.d_747006172 != '353358909' -- has not withdrawn consent")
    
    bq2_tbl_name <- gsub("_v1_JP", "", table) # remove the _v1_JP pattern from file name
-   write(sql, file = glue("{outputpath}/{tier}/bq2_{bq2_tbl_name}.sql"))
+   outputpath = "/Users/sansalerj/Desktop/bq2/bq2/original_bq2_query_gen/"
```

Comment:
This overwrites the outputpath that is parameterized at the top of the script with one that references your local filesystem. 

Again, this is not a big deal since we probably will move away from using this file anyway.